bundles/python/misc/examples.py

```diff
@@ -0,0 +1,252 @@
+#! /usr/bin/env python
+
+# Comments use '#'
+
+# Lists
+
+empty_list = []
+
+# Builtin Identifiers
+
+builtins = [ArithmeticError, AssertionError, AttributeError,
+BaseException, BufferError, BytesWarning, DeprecationWarning,
+EOFError, Ellipsis, EnvironmentError, Exception, False,
+FloatingPointError, FutureWarning, GeneratorExit, IOError,
+ImportError, ImportWarning, IndentationError, IndexError, KeyError,
+KeyboardInterrupt, LookupError, MemoryError, NameError, None,
+NotImplemented, NotImplementedError, OSError, OverflowError,
+PendingDeprecationWarning, ReferenceError, RuntimeError, RuntimeWarning,
+StandardError, StopIteration, SyntaxError, SyntaxWarning, SystemError,
+SystemExit, TabError, True, TypeError, UnboundLocalError,
+UnicodeDecodeError, UnicodeEncodeError, UnicodeError,
+UnicodeTranslateError, UnicodeWarning, UserWarning, ValueError,
+Warning, ZeroDivisionError, __debug__, __doc__, __import__,
+__name__, __package__, abs, all, any, apply, basestring, bin,
+bool, buffer, bytearray, bytes, callable, chr, classmethod, cmp,
+coerce, compile, complex, copyright, credits, delattr, dict,
+dir, divmod, enumerate, eval, execfile, exit, file, filter,
+float, format, frozenset, getattr, globals, hasattr, hash, help,
+hex, id, input, int, intern, isinstance, issubclass, iter,
+len, license, list, locals, long, map, max, memoryview, min,
+next, object, oct, open, ord, pow, property, quit,
+range, raw_input, reduce, reload, repr, reversed, round, set,
+setattr, slice, sorted, staticmethod, str, sum, super, tuple,
+type, unichr, unicode, vars, xrange, zip]
+
+# Identifiers
+if False:
+  idents = [a, b, cde, fgh1, _blah, _foo_abc013, UPPER_CASE, Mixed8347df_]
+
+
+# Integers
+
+ints = [1, 2, 3, 4, 5, 38474, 13313]
+longs = [1L, 30l, 347388472349449848L]
+
+# Floats
+
+floats = [1.0, 2.0, 3.456, 3.234e123, 1342343e199394]
+
+# Complex
+
+complex = [1j, 2j, 3 + 4j, 7.18384 + 43.3847j]
+
+# Strings
+
+sq_strings = ['single', 'quoted', 'encloses "double" quotes', 'backlash escapes \' ']
+dq_strings = ["single", "quoted", "encloses 'single' quotes", "backlash escapes \" "]
+
+triple_sq_string = '''can span
+multiple lines easily
+single 'quotes' ok - no escaping needed
+double "quotes" ok too!
+'''
+
+triple_dq_strings = """can span
+multiple lines easily
+single 'quotes' ok - no escaping needed
+double "quotes" ok too!
+"""
+
+adjascent_strings_auto_concat = "hello " "world"
+same_as_above = "hello""world"
+
+if False:
+  # Dictionaries
+  d = {a: b, c: d, "s": 1234 }
```

Comment:
Yes, for kwargs I was referring to the call site. If you don't think it makes sense than I'll happily defer to your judgement. The `key` style is not strictly defined (an as an aside it would probably be a good idea to go through all default styles at some point and perhaps restructure these). But note that in both Lua and Moonscript it's possible to use an expression as a table key, so by that logic the lexing for those languages is also odd.